charts/reform-scan-blob-router/values.yaml

```diff
@@ -27,6 +27,8 @@ java:
     DELETE_DISPATCHED_FILES_CRON: "0 0/10 * * * *"
     HANDLE_REJECTED_FILES_CRON: "0/30 * * * * *"
     REJECT_DUPLICATES_CRON: "0/30 * * * * *"
+    CHECK_NEW_ENVELOPES_CRON: "0 55 0 * * MON-FRI" # every hour at 55 minutes on weekdays
```

Comment:
The alert (for checking the logs) will run at every hour so the envelope task can run 5 minutes before so it will be reflected in the app insights. shall I change it to run on every day?

Reply:
we want to check only during business hours I think